spellbook/dbt_project.yml

```diff
@@ -128,5 +131,26 @@ seeds:
             block_time: timestamp
             tx_hash: string
             amount: string
-
+    ens:
+      ethereum:
```

Comment:
Here, if you had ethereum then you need to make sure the files are under ens/ethereum/filename.csv (they are currently under ens/filename.csv)

Reply:
So you either need to remove ethereum from the dbt_project.yml file or to add a folder in which you place the files